glossary.md

```diff
@@ -0,0 +1,66 @@
+# Glossary
+### CC Block
+Contains one keyblock and possibly one microblock
+
+### Keyblock
+A placeholder block indicating the leader that should produce microblocks in this generation.
+
+### Microblock
+A block with transactions.
+
+### Epoch
+Period of time represented in blocks during which validators are pre-selected to produce blocks and after which the state of the CC can be pinned to the PC.
```

Comment:
Possibly call this a "CC Epoch then".
There's also epoch for PC, but that differs in definition